hera_qm/tests/test_xrfi.py

```diff
@@ -217,9 +228,64 @@ def test_detrend_medfilt(self):
         ans = np.genfromtxt(ans_fn, dtype=np.complex)
         nt.assert_true(np.allclose(ans, dm))
 
+    def test_detrend_medfilt_3d_error(self):
         # Test error when wrong dimensions are passed
         nt.assert_raises(ValueError, xrfi.detrend_medfilt, np.ones((5, 4, 3)))
 
+    def test_detrend_meanfilt(self):
+        # make fake data
+        data = np.zeros((self.size, self.size))
+        for i in range(data.shape[1]):
+            data[:, i] = i * np.ones_like(data[:, i])
+        # run detrend medfilt
+        Kt = 8
+        Kf = 8
+        dm = xrfi.detrend_meanfilt(data, Kt=Kt, Kf=Kf)
+
+        # read in "answer" array
+        # this is output that corresponds to self.size==100, Kt==101, Kf==101
+        ans_fn = os.path.join(DATA_PATH, 'test_detrend_meanfilt_ans.txt')
```

Comment:
Why do you have to do this? Can't we concoct a test that we know the answer analytically?

Reply:
In theory. It got to be a pain because of the mirroring. I opted to take the same route that Paul did on other tests and just do it once and the test makes sure you don't break things when updating stuff.